include/common/serial_reader.h

```diff
@@ -55,21 +57,21 @@ class SerialReader {
             }
             tcflush(fileDescriptor, TCIFLUSH);
             return serialString;
-        }
+        /*}
         catch(const char *error) {
             std::cout << "[SerialReader:readData()] : " << error << std::endl;
         }
-        return "";
+        return "";*/
```

Comment:
I know this is technically my fault! But would you mind dropping the commented lines here...